specification/reservations/resource-manager/Microsoft.Capacity/preview/2020-10-01-preview/reservations.json

```diff
@@ -322,6 +322,49 @@
         }
       }
     },
+    "/providers/Microsoft.Capacity/reservationOrders/{reservationOrderId}/changeDirectory": {
+      "post": {
+        "summary": "Change direcotory of `ReservationOrder`.",
+        "description": "Change directory (tenant) of `ReservationOrder` and all `Reservation` under it to specified tenant id",
+        "operationId": "ReservationOrder_ChangeDirectory",
+        "x-ms-examples": {
+          "Split": {
+            "$ref": "./examples/ChangeDirectoryReservationOrder.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ReservationOrderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ChangeDirectoryParameter"
+          }
+        ],
+        "tags": [
+          "ReservationOrder, ChangeDirectory"
+        ],
+        "responses": {
+          "200": {
+            "description": "List of `Reservation`s created after the split operation.",
+            "schema": {
+              "type": "array",
```

Comment:
as per your example this should not be an array

Reply:
will fix